proposals/stage-0/MEMBERSHIP_PROGRAM/README.md

```diff
@@ -0,0 +1,61 @@
+# Individual Membership Representation Proposal
+>  Stage 0
+
+## Champion
+
+[Sara Chipps](https://github.com/sarajo)
+ 
+## Description
+
+The goals of the Individual Membership Program are to help bring awareness about both large and small projects to possible contributors. A way for individuals to show support and be a part of the OpenJS foundation, as well as providing an opportunity for community to collaborate with the other members. 
+
+While one goal of this group is introducing members to projects and helping the projects find the support they need; the membership program will also be an opportunity to welcome an inclusive group of coders from around the world that love the projects that are part of the OpenJS foundation. This also brings in new perspectives and feedback from the vast community of JavaScript developers.
+
+Additionally, Individual membership should help people understand that their projects too can get support from the foundation, and the OpenJS foundation is their foundation. 
+A tertiary goal is to establish a pool from which an ambassador program can be started. Recognizing people that evangelize and recruit for OpenJS Foundation projects. 
+
+## Membership Benefits
```

Comment:
I would love to explore more project related benefits. For example the ability to have a way to directly contact the maintainers of each individual project (if they sign up for it). While it would not be desirable to make projects exclusive to non OpenJS Members there might be room for providing some benefits exclusively for them.

I don't have any concrete ideas yet but will think about it.

Reply:
I love this idea, I think it's worth discussing. It seems like a great benefit.